Warnings from Visual Studio 2022 around clipboard

I just noticed the compile is mentioning this:

> plan-b\src\PlanViewer.App\Controls\QuerySessionControl.axaml.cs(123,30,123,54): warning CS0618: 'IClipboard.GetTextAsync()' is obsolete: 'Use ClipboardExtensions.TryGetTextAsync instead'

This issue is to investigate and remediate that issue.

_Migrated from plan-b#8_

--- a/src/PlanViewer.App/Controls/QuerySessionControl.axaml.cs
+++ b/src/PlanViewer.App/Controls/QuerySessionControl.axaml.cs
@@ -8,6 +8,7 @@ using System.Threading.Tasks;
 using System.Xml.Linq;
 using Avalonia.Controls;
 using Avalonia.Input;
+using Avalonia.Input.Platform;
 using Avalonia.Interactivity;
 using Avalonia.Layout;
 using Avalonia.Media;
@@ -120,7 +121,7 @@ public partial class QuerySessionControl : UserControl
         {
             var clipboard = TopLevel.GetTopLevel(this)?.Clipboard;
             if (clipboard == null) return;
-            var text = await clipboard.GetTextAsync();
+            var text = await clipboard.TryGetTextAsync();
             if (string.IsNullOrEmpty(text)) return;
             QueryEditor.TextArea.PerformTextInput(text);
         };

--- a/src/PlanViewer.App/MainWindow.axaml.cs
+++ b/src/PlanViewer.App/MainWindow.axaml.cs
@@ -9,6 +9,7 @@ using System.Threading.Tasks;
 using System.Xml.Linq;
 using Avalonia.Controls;
 using Avalonia.Input;
+using Avalonia.Input.Platform;
 using Avalonia.Interactivity;
 using Avalonia.Layout;
 using Avalonia.Media;
@@ -370,7 +371,7 @@ public partial class MainWindow : Window
         var clipboard = this.Clipboard;
         if (clipboard == null) return;
 
-        var xml = await clipboard.GetTextAsync();
+        var xml = await clipboard.TryGetTextAsync();
         if (string.IsNullOrWhiteSpace(xml))
         {
             ShowError("The clipboard does not contain any text.");

